cilium/cmd/helpers.go

```diff
@@ -79,3 +82,47 @@ func requireServiceID(cmd *cobra.Command, args []string) {
 		Usagef(cmd, "Empty service id argument")
 	}
 }
+
+//JSONOutput flag to enable json outout
+var JSONOutput bool
+
+//JSONPath flag to enable Json path support on commands
+var JSONPath string
```

Comment:
This variable doesn't need to be public, you can access it right from `DumpJSON`